app/uk/gov/hmrc/agentsubscription/controllers/RegistrationController.scala

```diff
@@ -34,12 +35,9 @@ class RegistrationController @Inject()(val desConnector: DesConnector, val authC
   def getRegistration(utr: String, postcode: String) = Action.async { implicit request =>
     ensureAuthenticated {
       desConnector.getRegistration(utr) map {
-        //TODO whitespace / case insensitive postcode matching?
-        case Some(desRegistrationResponse) => desRegistrationResponse.postalCode.contains(postcode) match {
-          case true => Ok(toJson(RegistrationDetails(desRegistrationResponse.isAnASAgent)))
-          case false => NotFound
-        }
-        case _ => NotFound
+        case Some(desRegistrationResponse) if postcodesMatch(desRegistrationResponse.postalCode, postcode) => Ok(toJson(RegistrationDetails(desRegistrationResponse.isAnASAgent)))
```

Comment:
this might've been what you meant earlier, if so sorry I didn't understand what you meant then!